src/app/store/comicses.js

```diff
@@ -0,0 +1,48 @@
+import api from "../../api";
```

Comment:
I would name it `Api` as it's a class. And even more. Maybe it will make sense to get rid of `api` folder, replace it with `utils` and create Api.js file with module instead of class?